lib/tests/streamlit/checkbox_test.py

```diff
@@ -66,11 +66,11 @@ def test_inside_form(self):
 
         # Calling `with` will invoke `__exit__` on `DeltaGenerator`
```

Comment:
We should also clean these comments, as no submit button will be created on `__exit__` anymore.